app/index.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
```

Comment:
So you like to spell IE lowercase and doctype uppercase. Inverse of what I think is logical.